docs/getting-started/policies/index.rst

```diff
@@ -195,7 +195,33 @@ method to interact with this directly. For example:
   oso.load_str("role(user, role_name) if user.role = role_name;")
 
   alice = User("alice", "accountant")
-  assert oso.query_rule("role", alice, "accountant")
+  assert list(oso.query_rule("role", alice, "accountant"))
```

Comment:
What is this `assert` supposed to be conveying to readers? That `query_rule` returns anything?

Reply:
That the rule succeeds.

Reply:
Not ideal syntax for this